policyengine_us/variables/gov/states/mi/tax/income/credits/home_heating/mi_heating_credit.py

```diff
@@ -0,0 +1,52 @@
+from policyengine_us.model_api import *
+
+
+class mi_heating_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan household heating cost credit"
+    defined_for = StateCode.MI
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.michigan.gov/taxes/iit/accordion/credits/table-a-2022-home-heating-credit-mi-1040cr-7-standard-allowance"
+        "http://www.legislature.mi.gov/(S(keapvg1h2vndkn25rtmpyyse))/mileg.aspx?page=getObject&objectName=mcl-206-527a"
+        )
+    def formula(tax_unit, period, parameters):
```

Comment:
need to add that not eligible if dependent on another return (use "dsi" variable)

Reply:
also if you are a student -> cant claim credit

Reply:
still open